autoload/neomake/makers/ft/cs.vim

```diff
@@ -0,0 +1,14 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#cs#EnabledMakers()
+    if neomake#utils#Exists('mcs')
```

Comment:
You can remove this check. It's only needed in special circumstances. In other words, just returning `['mcs']` from this function unconditionally is fine.